app/src/store/views/closeAccountView.ts

```diff
@@ -75,10 +81,11 @@ export default class CloseAccountView {
   async closeAccount() {
     if (!this.isValid) return;
 
+    this.setLoading(true);
     try {
       await this._store.orderStore.cancelAllOrders();
     } catch (error) {
-      this._store.appView.handleError(error, 'Unable to cancel all open orders');
+      this._store.appView.handleError(error as Error, 'Unable to cancel all open orders');
```

Comment:
Do we need to call `this.setLoading(false)` in case of an error? Or is that handled by `handleError()`?

Reply:
Ah, good catch. Thx for spotting that. I fixed this in https://github.com/lightninglabs/lightning-terminal/pull/375/commits/88ebfc371d84829a60a1da2c3bc152c34e699b00